docs/docsite/rst/dev_guide/platforms/aws_guidelines.rst

```diff
@@ -387,27 +387,18 @@ The combination of these two approaches is then:
            module.fail_json_aws(e, msg="Could not describe some resource")
 
 
-If the underlying ``describe_some_resources`` API call throws a ``ResourceNotFound``
-exception, ``AWSRetry`` takes this as a cue to retry until it's not thrown (this
-is so that when creating a resource, we can just retry until it exists).
-
-To handle authorization failures or parameter validation errors in
-``describe_some_resource_with_backoff``, where we just want to return ``None`` if
-the resource doesn't exist and not retry, we need:
+Prior to Ansible 2.10 if the underlying ``describe_some_resources`` API call threw
+a ``ResourceNotFound`` exception, ``AWSRetry`` would take this as a cue to retry until
+it's not thrown (this is so that when creating a resource, we can just retry until it
+exists).  This default was changed and it's now necessary to explicitly request
```

Comment:
```suggestion
exists).  This default was changed and it is now necessary to explicitly request
```

@tremble Do you think it would be better to specify a particular version of the Ansible or AWS collection here?

Reply:
I think it pre-dates the start of the collection...